benchmarks/shelley3pools/tx-generator-script.json

```diff
@@ -2,27 +2,33 @@
     { "setInitCooldown": 10 },
     { "setNumberOfInputsPerTx": 1 },
     { "setNumberOfOutputsPerTx": 1 },
-    { "setNumberOfTxs": 500 },
+    { "setNumberOfTxs": 5000 },
     { "setTPSRate": 10 },
     { "setTxAdditionalSize": 0 },
     { "setFee": 0 },
     { "setTTL": 1000000 },
     { "startProtocol": "configuration/configuration-generator.yaml" },
-    { "setEra": "MaryEra" },
+    { "setEra": "AllegraEra" },
     { "setLocalSocket": "logs/sockets/1" },
     { "readSigningKey": "pass-partout", "filePath": "configuration/genesis-shelley/utxo-keys/utxo1.skey" },
     { "secureGenesisFund": "genFund", "genesisKey": "pass-partout", "fundKey": "pass-partout" },
     { "delay": null },
     { "splitFund": [ "fund1", "fund2", "fund3","fund4" ], "sourceFund": "genFund", "newKey": "pass-partout" },
     { "delay": null },
-    { "splitFundToList": "fundList", "sourceFund": "fund1", "newKey": "pass-partout" },
-    { "prepareTxList": "txlist", "newKey": "pass-partout", "fundList": "fundList" },
+    { "splitFundToList": "allegraFunds", "sourceFund": "fund1", "newKey": "pass-partout" },
```

Comment:
What is the difference between `splitFund` and `splitFundToList`?

Reply:
It is hardly visible or understandable from looking at the JSON encoding.
Both functions have different types:
`splitFund` in the example binds the identifiers `fund1` `fund2` `fund3` and `fund4` to a value of type `Fund`.
`splitFundList` binds the identifier `fundList` to type `[Fund]`.length of the list is implicit from `txCount`.

Reply:
That is confusing..  Maybe it'd help to have a document that would explain all commands -- their meaning, and motivation.